iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/Bundle/DataBrokerProtectionBundleExtension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  DataBrokerProtectionBundleExtension.swift
```

Comment:
I'm not sure why I'm getting these SwiftLint header errors? I figured I must have missed a step making the new package, but as far as I can tell the setup is the same as the macOS one